borsh-derive/src/internals/schema/enums/snapshots/borsh_discriminant_true.snap

```diff
@@ -42,21 +42,16 @@ impl borsh::BorshSchema for X {
         <XD as borsh::BorshSchema>::add_definitions_recursively(definitions);
         <XE as borsh::BorshSchema>::add_definitions_recursively(definitions);
         <XF as borsh::BorshSchema>::add_definitions_recursively(definitions);
-        let discriminant_0: u8 = 0;
-        let discriminant_1: u8 = 20;
```

Comment:
@mina86 these were added intentionally, after [commit](https://github.com/near/borsh-rs/pull/232/commits/2c1aa85c7dbda0b219f1d3576ada2ba821a746ba). 

So that this case

```rust
    #[allow(unused)]
    #[derive(BorshSchema)]
    #[borsh(use_discriminant = true)]
    #[repr(i16)]
    enum XY {
        A,
        B = 0x100,
        C,
        D(u32, u32),
        E = 10,
        F(u64),
    }
```
becomes an error:
```bash
 1  error: literal out of range for `u8`
    --> borsh/tests/test_enum_discriminants.rs:220:13
     |
 220 |         B = 0x100,
     |             ^^^^^
     |
     = note: the literal `0x100` (decimal `256`) does not fit into the type `u8` and will become `0u8`
     = help: consider using the type `u16` instead
     = note: `#[deny(overflowing_literals)]` on by default
```

Reply:
I believe this can be handled with `u8::from(...)`.